packages/checkbox/checkbox.scss

```diff
@@ -0,0 +1,89 @@
+@import "~@fremtind/jkl-core/build/scss/variables/colors";
+@import "~@fremtind/jkl-core/build/scss/functions";
+
+$checkbox-size: 1.5rem;
+
+.jkl-checkbox {
+    align-items: center;
+    cursor: pointer;
+    display: inline-flex;
+    padding: rem(5px);
+
+    &__input {
+        -webkit-appearance: none;
+        background-color: $snøhvit;
+        border: solid rem(1px) $svart;
+        height: $checkbox-size;
+        margin-right: 0.5rem;
+        position: relative;
+        transition: background-color 200ms ease, transform 150ms ease;
+        vertical-align: middle;
+        width: $checkbox-size;
+
+        &:focus {
+            outline-offset: rem(2px);
```

Comment:
Not supported in IE ☹️

Reply:
![hello](https://media3.giphy.com/media/k61nOBRRBMxva/giphy.gif)